modules/key-vault/vault/.test/common/main.test.bicep

```diff
@@ -64,42 +64,14 @@ module testDeployment '../../main.bicep' = {
   params: {
     enableDefaultTelemetry: enableDefaultTelemetry
     name: '${namePrefix}${serviceShort}002'
-    accessPolicies: [
```

Comment:
I'd veto against this change (we have to discuss this) as you're right that access policies are not PSRule/WAF aligned - but - without them here, we don't test the child resource.

My suggestion: Let's create a copy of the min test and call it 'access policies', and the access policies in there (and exclude the file from the PSRule test), and perform the change in the common test as per your suggestion.

Reply:
Already implemented